tutorials/tts/FastPitch_GermanTTS_Training.ipynb

```diff
@@ -638,60 +302,64 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c60ed72a",
-   "metadata": {},
+   "id": "0ab8b471",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
    "outputs": [],
    "source": [
     "!wandb login #paste_wandb_apikey_here"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "51ec332e",
+   "id": "d4364261",
    "metadata": {},
    "source": [
-    "Now we are ready for training our model! Let's try to train FastPitch. Paste the PITCH_MEAN and PITCH_STD from previous steps here."
+    "Now we are ready for training our model! Let's try to train FastPitch. Paste the `PITCH_MEAN` and `PITCH_STD` from previous steps here."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "4ad43d30",
+   "id": "0e8e0cd7",
    "metadata": {},
    "outputs": [],
    "source": [
-    "!(cd NeMoGermanTTS && CUDA_VISIBLE_DEVICES=0 python fastpitch.py --config-path . --config-name fastpitch_align_22050 \\\n",
+    "!(cd NeMoGermanTTS && CUDA_VISIBLE_DEVICES=0 python fastpitch.py --config-path . --config-name fastpitch_align_22050_grapheme \\\n",
     "    model.train_ds.dataloader_params.batch_size=32 \\\n",
     "    model.validation_ds.dataloader_params.batch_size=32 \\\n",
-    "    train_dataset=../DataGermanTTS/thorsten-de/train_manifest_phonemes.json \\\n",
-    "    validation_datasets=../DataGermanTTS/thorsten-de/val_manifest_phonemes.json \\\n",
-    "    sup_data_path=../DataGermanTTS/thorsten-de/phonemes/ \\\n",
-    "    whitelist_path=./whitelist.tsv \\\n",
+    "    train_dataset=train_manifest_text_normed.json \\\n",
+    "    validation_datasets=val_manifest_text_normed.json \\\n",
+    "    sup_data_path=sup_data \\\n",
+    "    whitelist_path=whitelist.tsv \\\n",
     "    exp_manager.exp_dir=resultGermanTTS \\\n",
     "    trainer.max_epochs=1 \\\n",
     "    trainer.check_val_every_n_epoch=1 \\\n",
-    "    pitch_mean=#paste_pitch_mean_here \\\n",
-    "    pitch_std=#paste_pitch_std_here \\\n",
+    "    pitch_mean=126.70349884033203 \\\n",
+    "    pitch_std=31.456707000732422 \\\n",
```

Comment:
Why change this? Do we not want to "force" users to pay some attention and paste in the stats they've retrieved themselves?

Reply:
Yes, we want users to override these variables based on what they obtained in the previous step. I added these values as a reference in order to make our tutorial run automatically. I would add more guide in https://github.com/NVIDIA/NeMo/pull/6051/files#diff-aeb570ef44d8aab9809f9c3c1b249f3ed892179c1de877a49f4fa9544d657c2aR321